erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -663,9 +664,29 @@ def make_stock_entry(work_order_id, purpose, qty=None):
 			additional_costs = get_additional_costs(work_order, fg_qty=stock_entry.fg_completed_qty)
 			stock_entry.set("additional_costs", additional_costs)
 
+		add_additionl_cost(stock_entry, work_order)
```

Comment:
This function should be in BOM and also called from `get_items` method in Stock Entry.